src/biokbase/narrative/tests/test_jobcomm.py

```diff
@@ -193,7 +195,7 @@ def test_lookup_job_states__1_ok(self):
         "biokbase.narrative.jobs.jobcomm.jobmanager.clients.get", get_mock_client
     )
     def test_lookup_job_states__2_ok(self):
-        job_id_list = [JOB_COMPLETED, JOB_ERROR]
+        job_id_list = [JOB_COMPLETED, BATCH_PARENT]
```

Comment:
Batch parent is a little funny so try to sprinkle that in more :/

Reply:
Good plan.